pkg/cmd/server/origin/master.go

```diff
@@ -44,7 +44,6 @@ func (c *MasterConfig) newOpenshiftAPIConfig(kubeAPIServerConfig apiserver.Confi
 	// most of the config actually remains the same.  We only need to mess with a couple items
 	genericConfig := kubeAPIServerConfig
 	// TODO try to stop special casing these.  We should all agree on them.
-	genericConfig.AdmissionControl = c.AdmissionControl
```

Comment:
Remove the TODO, since it's relevant to the piece of code you're removing.

Reply:
> Remove the TODO, since it's relevant to the piece of code you're removing.

There is still one left.  We're winning though!